lib/Controller/ViewController.php

```diff
@@ -114,6 +119,7 @@ public function index():TemplateResponse {
 		$this->initialStateService->provideInitialState('show_tasks', $showTasks);
 		$this->initialStateService->provideInitialState('tasks_enabled', $tasksEnabled);
 		$this->initialStateService->provideInitialState('hide_event_export', $hideEventExport);
+		$this->initialStateService->provideInitialState('force_event_alarm_type', $forceEventAlarmType);
```

Comment:
Based om the docs is null not allowed as key:
https://github.com/nextcloud/server/blob/3e067edcbe80797a70118f613ac4b752ce512992/lib/public/AppFramework/Services/IInitialState.php#L45

Reply:
Yes, as mentioned in #4051, this causes log entries.